sql/common/init.sql

```diff
@@ -0,0 +1,13 @@
+DO $$
+DECLARE 
+    skip BOOLEAN;
+BEGIN 
+    SELECT current_setting('timescaledb.restoring', true) IS NOT DISTINCT FROM 'on' INTO skip;
```

Comment:
Maybe convert skip into do_setup or something to reduce the NOTs ? If you keep skip, make it lower case in if statement.